api/Serialization.ts

```diff
@@ -280,7 +280,6 @@ export module ContractSerializer {
                                     var value = enumType.enumValues[name];
                                     if (name.toLowerCase() === lowerCaseEnumName) {
                                         resultPart = value;
-                                        return resultPart;
```

Comment:
Could we change this to `break;` instead? That way we don't keep spinning on the loop

Reply:
makes sense.